app/templates/events/view.hbs

```diff
@@ -1,5 +1,4 @@
-{{#if (or (eq session.currentRouteName 'events.view.index') (eq session.currentRouteName 'events.view.export'))}}
-  {{#if (eq model.state 'draft')}}
```

Comment:
Revert this

Reply:
i don't get it .

> Revert this

I don't get it . please explain a bit.

Reply:
`{{#if (eq model.state 'draft')}}`
Let it be on next line only